modules/oe_whitelabel_helper/src/Plugin/field_group/FieldGroupFormatter/PatternFormatterBase.php

```diff
@@ -39,13 +32,16 @@ abstract class PatternFormatterBase extends FieldGroupFormatterBase implements C
    *   The plugin_id for the plugin instance.
    * @param array $plugin_definition
    *   The plugin implementation definition.
-   * @param \Drupal\ui_patterns\UiPatternsManager $patterns_manager
+   * @param \Drupal\ui_patterns\UiPatternsManager $patternsManager
    *   UI Patterns manager.
    */
-  public function __construct(array $configuration, string $plugin_id, array $plugin_definition, UiPatternsManager $patterns_manager) {
+  public function __construct(
+    array $configuration,
+    string $plugin_id,
+    array $plugin_definition,
+    protected UiPatternsManager $patternsManager,
+  ) {
     parent::__construct($plugin_id, $plugin_definition, $configuration['group'], $configuration['settings'], $configuration['label']);
-    $this->configuration = $configuration;
-    $this->patternsManager = $patterns_manager;
```

Comment:
For the `patternManager`, this is just replaced with constructor property promotion, so it will be ok for extending classes.
The only concern could be calling code using named argument syntax, because the parameter name changes from `$patterns_manager` to `$patternsManager`.
We can remove this change from the PR, I think I just had it somewhere and kept it in.

For the other change, indeed removing `$configuration` would break existing extending classes if they rely on this property. So bad idea, and should not be part of the PR.

In the end I would be ok removing both commits.